code/game/objects/items/mountable_frames/trophy_mount.dm

```diff
@@ -0,0 +1,140 @@
+/obj/item/mounted/frame/trophy_mount
+	name = "trophy mount"
+	desc = "A wooden trophy mount."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "trophy_mount"
+	w_class = W_CLASS_MEDIUM
+	flags = FPRINT
+	w_type = RECYK_WOOD
+	frame_material = /obj/item/stack/sheet/wood
+	sheets_refunded = 2
+	autoignition_temperature = AUTOIGNITION_WOOD
+	var/obj/item/held_item
+
+/obj/item/mounted/frame/trophy_mount/Destroy()
+	if(held_item)
+		qdel(held_item)
+		held_item = null
+	..()
+
+/obj/item/mounted/frame/trophy_mount/update_icon()
+	if(!held_item)
+		overlays.len = 0
+		name = initial(name)
+		desc = initial(desc)
+		return
+	name = "[held_item.name]"
+	desc = "\A [held_item] mounted on a wooden trophy mount for display."
+	var/datum/log/L = new
+	held_item.examine(L)
+	desc += "\n[L.log]"
+	qdel(L)
+	var/image/temp = new (src)
+	temp.appearance = held_item.appearance
+	temp.transform = matrix()
+	temp.dir = SOUTH
+	temp.plane = FLOAT_PLANE
+	if(istype(held_item, /obj/item/organ/external/head))	//not every item can be tailored to fit well, but heads get special consideration
+		temp.pixel_y = -8 * PIXEL_MULTIPLIER
+	overlays += temp.appearance
+
+/obj/item/mounted/frame/trophy_mount/attackby(obj/item/weapon/W, mob/user)
+	..()
+	if(iswrench(W))
+		return
```

Comment:
?

Reply:
Deconstructing via wrench is handled in the parent. If I didn't do this, it would cause application of a wrench to deconstruct the trophy mount, then insert the wrench into the now nonexistant trophy mount, nullspacing it.

Reply:
Ah.
We should really be using return values to prevent that kind of thing.